### PR TITLE
fix: array properties and some small bugs

### LINKED ISSUE #102: Subfields of an Array fields are not rendered
A field that has an array of objects is no longer rendering the fields of the object type in the array. For example, https://doc.crds.dev/github.com/crossplane/provider-aws/s3.aws.crossplane.io/BucketPolicy/v1alpha2#spec-forProvider-statements does not show all the fields in https://github.com/crossplane/provider-aws/blob/4e88c93f6c7c897d2842a125d0f1fe7f7a3243da/apis/s3/v1alpha2/bucketpolicy_types.go#L58

### PR DESCRIPTION
**Pull request by RichiCoder1**
Fixes a number of small render bugs:

- Array properties now will either display the primitive type (e.g. `[]string`) or the correct sub-properties.
- Adding some padding to the bottom of the page.
- Fixed an issue where the text of the new provider page was smushed into the nav bar.

Closes #102

### CODE REVIEW

**Comment by RichiCoder1** [comment]
localhost_5000_github com_kubernetes-sigs_cluster-api-provider-packet_crd projectcalico org_BGPConfiguration_v1(iPad Pro)

**Comment by RichiCoder1** [comment]
@hasheddan This should do it!

File: template/doc.html [hunk 1]
```diff
@@ -49,14 +51,6 @@
 
     function getDescription(schema) {
         let desc = schema.Description || '';
-        if (schema.Enum) {
-            desc += '\n' + `
-\`\`\`yaml
-# valid values
-${schema.Enum.map(e => `- '${e}'`).join('\n')}
-\`\`\``.trim()
```
Comment (hasheddan) [inline template/doc.html]: Does this mean Enum values will not be shown at all?

**Review by hasheddan — COMMENTED** [review]
Looks really good @RichiCoder1! One question below

File: template/doc.html (hunk 1 above)
Comment (hasheddan) [reply template/doc.html]: I think this works for now :) If we get feedback that folks would like them called out explicitly then we can add back. Thanks again!

**Review by hasheddan — APPROVED** [review]

**Pull request merged**